### PR TITLE
feat: Add commitment parameter to transactions methods

### PR DESCRIPTION
**Pull request by ookami-kb**

### CODE REVIEW

File: lib/src/solana_client.dart [hunk 1]
```diff
@@ -207,11 +207,17 @@ class SolanaClient {
   ///
   /// Providing [before] and [until] also moves the cursor to a more specific
   /// subset.
+  ///
+  /// For [commitment] parameter description see
+  /// https://docs.solana.com/developing/clients/jsonrpc-api#configuring-state-commitment
+  ///
+  /// [TxStatus.processed] is not supported as [commitment].
```
Comment (iharobCP) [inline lib/src/solana_client.dart:214]: What does this comment mean?

**Review by iharobCP — COMMENTED** [review]
Can you take the

File: lib/src/solana_client.dart (hunk 1 above)
Comment (ookami-kb) [reply lib/src/solana_client.dart:214]: https://docs.solana.com/developing/clients/jsonrpc-api#getconfirmedtransaction

> (optional) Commitment; "processed" is not supported. If parameter not provided, the default is "finalized".

Comment (iharobCP) [reply lib/src/solana_client.dart:214]: Ok, understood.

**Review by iharob — APPROVED** [review]

**Review by iharobCP — APPROVED** [review]

**Pull request merged**